spec/tags/truffle/methods_tags.txt

```diff
@@ -0,0 +1,5 @@
+slow:Public methods on BasicObject are the same as on MRI
+slow:Public methods on Object are the same as on MRI
+slow:Public methods on Enumerable are the same as on MRI
+slow:Public methods on Enumerator are the same as on MRI
+slow:Public methods on Numeric are the same as on MRI
```

Comment:
They need `ruby_exe` to ensure that no libraries are required or previous specs that could add extra methods on core classes/modules.